### PR TITLE
RW-9838 RW-10682 Make adding additional fields not break existing swagger client

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://precisionmedicineinitiative.atlassian.net/browse/RW-9838

Dependencies

- [x] https://github.com/DataBiosphere/cbas/pull/192
- [ ] [https://github.com/DataBiosphere/leonardo/pull/3588](Adopt new swagger in CBAS merged in Leo)
- [x] https://github.com/broadinstitute/rawls/pull/2493 merged
- [x] https://github.com/DataBiosphere/terra-workspace-data-service/pull/325
- [ ] Adopt new swagger in WDS merged in Leo
- [ ] Upgrade existing WDS and cromwell apps (try it out in dev first)

TODO:

- [x] Test it in BEE

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: codegen_java/gen_java_client.sh
Comment (davidangb) [inline codegen_java/gen_java_client.sh]: might need to be `--additional-properties=disallowAdditionalPropertiesIfNotPresent=false`? (change colon to equals)

I'm basing this comment off the example at https://openapi-generator.tech/docs/usage/#additional-properties

Comment (Qi77Qi) [reply codegen_java/gen_java_client.sh]: ah yea, you're totally right! I was looking at https://openapi-generator.tech/docs/usage/#additional-properties but just copy pasted from ur comment

**Review by davidangb — APPROVED** [review]
lgtm, assuming you test it and it actually fixes the issue!

**Comment by LizBaldo** [comment]
LGTM, did CBAS migrate over to using the new version of the client yet? They can point the the PR snapshot to make sure it is good to go before merging

**Review by LizBaldo — APPROVED** [review]

**Comment by Qi77Qi** [comment]
@LizBaldo yea good point...I'll coordinate with cromwell team on that

**Comment by rtitle** [comment]
The Leo change looks good -- just making the point that we'll need to update existing CBAS/WDS deployments so they won't break. Also we [exclude a lot of versions](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf#L258) from updates.

Might be good to do some update dryRuns in prod and compare what will be updated vs what will be skipped… and if we're okay breaking those apps.

**Comment by LizBaldo** [comment]
@Qi77Qi are you planning on merging this? Do you need support from IA?

**Comment by Qi77Qi** [comment]
@LizBaldo nope...I just need to address some other things before getting back to this

**Comment by LizBaldo** [comment]
No worries/rush :) I just wanted to make sure we were not blocking you

**Comment by Qi77Qi** [comment]
```
14:20:55 URL: GET https://clotho.broadinstitute.org:8200/v1/secret/dsde/dsp-techops/common/dspci-wb-gcr-service-account.json
14:20:55 Code: 400. Errors:
14:20:55 
```

jenkins retest

**Pull request merged**
